plugins/uicolor/dialogs/uicolor.js

```diff
@@ -122,11 +122,11 @@ CKEDITOR.dialog.add( 'uicolor', function( editor ) {
 	function setHighlightedColor( color ) {
 		if ( color ) {
 			$doc.getById( highlightedColorId ).setStyle( 'background-color', color );
-			$doc.getById( highlightedColorTextId ).setHtml( color );
+			$doc.getById( highlightedColorTextId ).setHtml( CKEDITOR.tools.htmlSafeByReview(CKEDITOR.tools.htmlEncode(color), 'Encoding does not break tests. The variable color here is only ever set to the html of elements created internally (in createColorTable and applyColorCell)') );
```

Comment:
Nice! Are we reasonably confident here that `htmlEncode` is a safe-enough "sanitizer"? If so, should we adopt this pattern more frequently in some of the other callsites in this PR? (Just an idea.)

Reply:
In this context, just removing the angle-brackets prevents escape, right? (Because the code isn't inserted as an attribute but inside another well-formed tag?)

If so, SGTM